exchange/src/client_manager/client_manager.cpp

```diff
@@ -3,58 +3,67 @@
 namespace nutc {
 namespace manager {
 
-inline bool
-ClientManager::user_exists(const std::string& uid) const
+bool
+ClientManager::user_exists(const std::string& id) const
 {
-    return clients.find(uid) != clients.end();
+    return clients.find(id) != clients.end();
 }
 
-inline bool
-ClientManager::user_holds_stock(const std::string& uid, const std::string& ticker) const
+void
+ClientManager::set_client_pid(const std::string& id, pid_t pid)
```

Comment:
also should probably go into the header

Reply:
It's only used once in program execution so I'm gonna keep it out for the sake of readability